src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -236,6 +240,10 @@ They can be configured through the settings file 'winget settings'.</value>
   <data name="InstallationDisclaimer2" xml:space="preserve">
     <value>Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.</value>
   </data>
+  <data name="InstallationDisclaimerMSStore" xml:space="preserve">

```

Comment:
>InstallationDisclaimerMSStore [](start = 14, length = 29)

If this is a placeholder, did you send mail to ask for an official message?

Reply:
I asked Kevin to give me one. We'll update later

---
In reply to: [455231560](https://github.com/microsoft/winget-cli/pull/493#discussion_r455231560) [](ancestors = 455231560)